package-lock.json

```diff
@@ -1052,11 +1053,103 @@
         "parsimmon": "^1.13.0"
       }
     },
+    "node_modules/@definitelytyped/typescript-packages": {
```

Comment:
I have no idea why this dep is being pulled in. Maybe I will extract the function.

Reply:
Sent https://github.com/microsoft/DefinitelyTyped-tools/pull/964